tests/modules/services/gpg-agent/default-homedir.nix

```diff
@@ -2,7 +2,15 @@
 
```

Comment:
To make the tests actually run for macOS, I think you would need to move the `./modules/services/gpg-agent` line in `tests/default.nix` to the common section.